osgbuild/git.py

```diff
@@ -290,7 +290,7 @@ def verify_correct_branch(package_dir, buildopts):
                 raise Error("""\
 Error: You must build into the HCC repo when building from
 a HCC git checkout.  You must switch git repos or build targets.""")
-        elif target.endswith('osg-upcoming'):
+        elif re.search(r"osg(?:-\d+\.\d+)?-upcoming$", target):
```

Comment:
I can't comment on the specific line but I think that the error message here needs updating https://github.com/opensciencegrid/osg-build/blob/66c18ca712c16bbdf7cb1f9437123a562c208598/osgbuild/git.py#L315

Reply:
Fixed.